packages/pn-commons/src/components/EmptyState.tsx

```diff
@@ -47,29 +49,25 @@ function EmptyState({
       }}
     >
       {FinalIcon && (
-        <FinalIcon sx={{ verticalAlign: 'middle', mr: '20px', mb: '2px', fontSize: "1.25rem" }} />
+        <FinalIcon sx={{ verticalAlign: 'middle', mr: '20px', mb: '2px', fontSize: '1.25rem' }} />
       )}
       <Typography variant="body2" sx={{ display: 'inline' }}>
         {emptyMessage}
       </Typography>
-      { emptyActionCallback && 
+      {emptyActionCallback && (
         <>
           &nbsp;
-          <Typography
-            color="primary"
-            variant="body2"
-            fontWeight={'bold'}
+          <ButtonNaked
             data-testid="callToActionFirst"
-            sx={{
-              cursor: 'pointer',
-              display: 'inline',
-            }}
             onClick={emptyActionCallback}
+            sx={{ buttonNakedInheritStyle }}
```

Comment:
This style maybe is useless, add only verticalAlign: 'unset' and check that mobile and desktop views are ok